groundzero_ddl/casev3.sql

```diff
@@ -98,6 +98,8 @@
        id  bigserial not null,
         batch_id uuid,
         batch_quantity int4,
+        correlation_id uuid not null,
```

Comment:
Since this is not null will you have to make sure there's no fulfilments waiting when you merge?

Reply:
yeah